internal/cluster/cluster.go

```diff
@@ -337,6 +340,8 @@ func (m *Manager) RefreshStatus(ctx context.Context, c *common.Cluster, db *gorm
 }
 
 func (m *Manager) refreshStatusInternal(ctx context.Context, c *common.Cluster, db *gorm.DB) (*common.Cluster, error) {
+
+	defer commonutils.MeasureOperationWithThresholdAndId("ClusterMonitoring-refreshStatusInternal", m.log, m.metricAPI, OperationThresholdInSec, c.ID.String())()
```

Comment:
I thought it wasn't a good practice to put a specific id into a metric.
Unless we're interested in filtering metrics for a specific cluster I feel like this will create a whole lot of data.

Didn't we have an issue with putting cluster ids into metrics before or am I misremembering @slaviered ?

Reply:
The idea is to find cluster/host that creates the issue. In regular case we shouldn't create the metric as threshold will not be reached.